routes/index.js

```diff
@@ -56,12 +57,18 @@ module.exports = function(app) {
     });
 
     // Return TwiML instuctions for the outbound call
-    app.post('/outbound', function(request, response) {
-        // We could use twilio.TwimlResponse, but Jade works too - here's how
-        // we would render a TwiML (XML) response using Jade
-        response.type('text/xml');
-        response.render('outbound');
-    });
-};
+    app.post('/outbound/:salesNumber', function(request, response) {
+        var salesNumber = request.params.salesNumber;
+        var twimlResponse = new twilio.TwimlResponse();
 
+        twimlResponse.say('Thanks for contacting our sales department. Our ' +
+                          'next available representative will take your call. '
+                          , {
+                            voice: 'alice',
+                          });
```

Comment:
Maybe this one is too short to go splitted apart on 3 lines :grey_question: 
```javascript
{voice: 'alice'}
```
Looks easy and good, not too long for a single line.